packages/material-ui-lab/src/Alert/Alert.js

```diff
@@ -0,0 +1,278 @@
+    '& $icon': {
+      color: theme.palette.warning.main,
+    },
+  },
+  /* Styles applied to the root element if `variant="outlined"` and `color="error"`. */
+  outlinedError: {
+    color:
+      theme.palette.type === 'light'
+        ? darken(theme.palette.error.main, 0.6)
+        : lighten(theme.palette.error.main, 0.6),
+    border: `1px solid ${theme.palette.error.main}`,
+    '& $icon': {
+      color: theme.palette.error.main,
+    },
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="success"`. */
+  filledSuccess: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.success.main,
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="info"`. */
+  filledInfo: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.info.main,
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="warning"`. */
+  filledWarning: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.warning.main,
+  },
+  /* Styles applied to the root element if `variant="filled"` and `color="error"`. */
+  filledError: {
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightMedium,
+    backgroundColor: theme.palette.error.main,
+  },
+  /* Styles applied to the icon wrapper element. */
+  icon: {
+    marginRight: 12,
+    padding: '7px 0',
+    display: 'flex',
+    fontSize: 22,
+    opacity: 0.9,
+  },
+  /* Styles applied to the message wrapper element. */
+  message: {
+    padding: '8px 0',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+  },
+  /* Styles applied to the action wrapper element if `action` is provided. */
+  action: {
+    display: 'flex',
+    alignItems: 'center',
+    marginLeft: 'auto',
+    paddingLeft: 16,
+    marginRight: -8,
+  },
+});
+
+const defaultIconMapping = {
+  success: <SuccessOutlinedIcon fontSize="inherit" />,
+  warning: <ReportProblemOutlinedIcon fontSize="inherit" />,
+  error: <ErrorOutlineIcon fontSize="inherit" />,
+  info: <InfoOutlinedIcon fontSize="inherit" />,
+};
+
+const Alert = React.forwardRef(function Alert(props, ref) {
+  const {
+    action,
+    children,
+    classes,
+    className,
+    closeText = 'Close',
+    color,
+    icon,
+    iconMapping = defaultIconMapping,
+    onClose,
+    role = 'alert',
+    severity = 'success',
+    variant = 'text',
+    ...other
+  } = props;
+
+  return (
+    <Paper
+      role={role}
+      square
+      elevation={0}
+      className={clsx(
+        classes.root,
+        classes[`${variant}${capitalize(color || severity)}`],
+        className,
+      )}
+      ref={ref}
+      {...other}
+    >
+      {icon !== false ? (
+        <div className={classes.icon}>
+          {icon || iconMapping[severity] || defaultIconMapping[severity]}
```

Comment:
```suggestion
          {icon || iconMapping[severity]}
```

Reply:
This logic is important for fallback.

Reply:
What am I missing? https://github.com/mui-org/material-ui/pull/18702/files/088cf5d8390fcf194f8cdc14248224f625444e59#diff-ba59df5f895c0b27cff62d4e541ac689R164